setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = ansible-navigator
-version = attr: ansible_navigator.__version__
+version = attr: ansible_navigator._version.version
```

Comment:
This doesn't get used when `setuptools-scm` is integrated. It's common to just drop it.
```suggestion
```
Keeping it would only create confusion because of the illusion that it's how the version is retrieved.